### PR TITLE
Upgrade to pysam 0.8.4

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #807

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Looks good, but lets put this one off until after we've released 0.2.0a1.

**Comment by jeromekelleher** [comment]
Can you update setup.py to include the new version also please?

File: ga4gh/datamodel/variants.py [hunk 1]
```diff
@@ -535,11 +535,7 @@ def buildMetadata(
         # NOTE: filters in not included in metadata unless needed
         for prefix, content in [("FORMAT", formats), ("INFO", infos)]:
             for contentKey, value in content:
-                attrs = dict(value.header.attrs)
-                # TODO: refactor description at next pysam release
-                # since description will be implemented as a member of
-                # VariantMetadata
-                description = attrs.get('Description', '').strip('"')
+                description = getattr(value, 'description', '').strip('"')
```
Comment (jeromekelleher) [inline ga4gh/datamodel/variants.py]: Why use getattr here rather than ``value.description``?

**Comment by dcolligan** [comment]
@jeromekelleher updated

**Comment by jeromekelleher** [comment]
Thanks @dcolligan, +1.

**Pull request merged**